crates/primitives/src/chain/spec.rs

```diff
@@ -212,6 +219,26 @@ pub struct ChainSpec {
     /// The block at which the deposit contract for PoS was deployed.
     #[serde(skip, default)]
     pub deposit_contract_deployment_block: Option<BlockNumber>,
+    /// EIP-1559 base fee change denominator, used in the base fee calculation
+    pub base_fee_change_denominator: u64,
+    /// EIP-1559 elasticity multiplier, used in the base fee calculation
+    pub elasticity_multiplier: u64,
+}
+
+impl Default for ChainSpec {
+    fn default() -> ChainSpec {
+        ChainSpec {
+            chain: Default::default(),
+            genesis_hash: Default::default(),
+            genesis: Default::default(),
+            paris_block_and_final_difficulty: Default::default(),
+            fork_timestamps: Default::default(),
+            hardforks: Default::default(),
+            deposit_contract_deployment_block: Default::default(),
```

Comment:
nit: `..Default::default()`

Reply:
I originally tried to use ..Default::default() as follows:

```
 fn default() -> ChainSpec {
        ChainSpec {
            base_fee_change_denominator: EIP1559_DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR,
            elasticity_multiplier: EIP1559_DEFAULT_ELASTICITY_MULTIPLIER,
            ..Default::default()
        }
    }
```
but the compiler gives a scary warning:

```
warning: function cannot return without recursing
   --> crates/primitives/src/chain/spec.rs:229:5
    |
229 |     fn default() -> ChainSpec {
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^ cannot return without recursing
...
233 |             ..Default::default()
    |               ------------------ recursive call site
```

I'm a rust n00b but that seemed worrisome?